cloudfront/wellcomelibrary.org/edge-lambda/src/wellcomeLibraryPassthru.test.ts

```diff
@@ -34,14 +34,17 @@ test('http requests are redirected to https', () => {
   );
 });
 
-test('rewrites the host header if it exists', async () => {
-  const request = testRequest('/', undefined, {
+test('redirects wellcomelibrary.org passthrus to wellcomecollection.org', async () => {
+  const request = testRequest('/unknown', undefined, {
     host: [{ key: 'host', value: 'www.wellcomelibrary.org' }],
   });
 
   const originRequest = await origin.requestHandler(request, {} as Context);
 
   expect(originRequest.headers).toStrictEqual({
-    host: [{ key: 'host', value: 'wellcomelibrary.org' }],
+    location: [{ key: 'Location', value: 'https://wellcomecollection.org/' }],
+    'access-control-allow-origin': [
+      { key: 'Access-Control-Allow-Origin', value: '*' },
+    ],
```

Comment:
Do you want a test here that we return a 302 response? You could return Location headers on a 200 OK.

Reply:
Nice idea, just did it.